common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -279,4 +279,31 @@ d_burning_blade = {
 		color1 = "red"
```

Comment:
> Shadowmoon

But this title doesn't exist.

P.S. I see it's used by Ner'zhul's dynasty, but you should mention it in the changelog.

Reply:
edited